app/models/event.rb

```diff
@@ -40,4 +40,9 @@ def self.upcoming
   def limit_newbies
     limit_total - limit_verified
   end
+
+  def limit_percent
```

Comment:
I am in doubt about this method should be in model if it required only for views :(